docs/runtimes/functions.md

```diff
@@ -1,27 +1,73 @@
 (Functions)=
-# Functions
+# Functions 
 
-All the executions in MLRun are based on **Serverless functions**. The functions allow specifying code and 
-all the operational aspects (image, required packages, [cpu/mem/gpu resources](../runtimes/configuring-job-resources.html#cpu-gpu-and-memory-limits-for-user-jobs), [storage](../runtimes/function-storage.html), environment, etc.). 
-The [different function runtimes](../concepts/functions-overview.html) take care of automatically transforming the code and spec to fully 
-managed and elastic services over Kubernetes, which saves significant operational overhead, 
-addresses scalability and reduces infrastructure costs.
+All the executions in MLRun are based on serverless functions. Functions are essentially Python objects that know how to run locally or on a Kubernetes cluster. 
```

Comment:
```suggestion
All the executions in MLRun are based on serverless functions. Functions are essentially Python code that can be executed locally or on a Kubernetes cluster. 
```

I don't like the "objects that know how to run..." idiom. The entire idea is that the code doesn't need to know anything about where it runs.

Reply:
updated